FlaxEditor/Options/GeneralOptions.cs

```diff
@@ -59,5 +59,13 @@ public sealed class GeneralOptions
         [DefaultValue(true)]
         [EditorDisplay("Analytics"), EditorOrder(1000), Tooltip("Enables or disables anonymous editor analytics service used to improve editor experience and the quality")]
         public bool EnableEditorAnalytics { get; set; } = true;
+        
+        [DefaultValue(true)]
+        [EditorDisplay("Auto Saves", "Enable Auto Saves"), EditorOrder(1000), Tooltip("Enables or disables auto saves")]
+        public bool EnabbleAutoSaves { get; set; } = true;
```

Comment:
I would suggest adding comment to match the general style, same in the property below.

Reply:
Typo in `EnabbleAutoSaves` (double `b`).